Firestore/tests/Unit/ValueMapperTest.php

```diff
@@ -192,9 +191,11 @@ public function decodedValues()
         $blobValue = 'hello world';
         $blob = new Blob($blobValue);
 
-        $now = (string) time();
-        $nanos = 10;
-        $timestamp = new Timestamp(\DateTimeImmutable::createFromFormat('U', (string) $now), $nanos);
+        $datetime = \DateTimeImmutable::createFromFormat('U.u', microtime(true));
+        $now = (string) $datetime->format('U');
+        $micros = intval($datetime->format('u'));
```

Comment:
I was a little hasty on hitting approve, could we please also update these instances?

Reply:
of course

Reply:
Thanks @DiogoRMax!